src/components/drag_and_drop/_draggable.scss

```diff
@@ -13,26 +12,31 @@
   }
 
   &.euiDraggable--hasCustomDragHandle > .euiDraggable__item [data-react-beautiful-dnd-drag-handle] {
-    border: 1px solid transparent;
+    // What was the reason for the transparent border?
```

Comment:
Now that it uses `euiFocusRing`, there is no reason for it. Before, there was a size change in the handle when the border was added.

Reply:
Cool, feel free to remove the comment then.